Description: lxc-debian: specify isc-dhcp-server in package list
 dhcp3-client does not exist in testing.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1052972
Forwarded: yes

Index: lxc/templates/lxc-debian.in
===================================================================
--- lxc.orig/templates/lxc-debian.in	2012-09-21 13:34:12.247622000 -0500
+++ lxc/templates/lxc-debian.in	2012-09-21 13:34:45.480764561 -0500
@@ -110,7 +110,7 @@
 locales,\
 libui-dialog-perl,\
 dialog,\
-dhcp3-client,\
+isc-dhcp-client,\
 netbase,\
 net-tools,\
 iproute,\
